AlphaWallet/BuyCrypto/SelectServiceToBuyCryptoCoordinator.swift

```diff
@@ -72,7 +72,8 @@ class SelectServiceToBuyCryptoCoordinator: Coordinator {
         } else if actions.count == 1 {
             completion(.service(actions[0].service))
         } else {
-            let alertController = UIAlertController(title: nil, message: .none, preferredStyle: .actionSheet)
+            let preferredStyle: UIAlertController.Style = UIDevice.current.userInterfaceIdiom == .pad ? .alert : .actionSheet
+            let alertController = UIAlertController(title: nil, message: nil, preferredStyle: preferredStyle)
```

Comment:
@oa-s Should we extract this into a short function and use it where it's needed (or not) and be easier to discover if iPads are handled correct or not? Make an issue or something if so?